src/org/labkey/test/tests/AdminConsoleTest.java

```diff
@@ -51,6 +59,74 @@ public String getProjectName()
     }
 
     @Test
+    public void testServerHttpHeaderSetting()
+    {
+        goToAdminConsole().clickSiteSettings();
+        waitForElement(Locator.name("includeServerHttpHeader"));
+        WebElement checkbox = Locator.checkboxByName("includeServerHttpHeader").findElement(getDriver());
+
+        boolean originalValue = "true".equals(checkbox.getAttribute("checked"));
```

Comment:
"checked" is a valueless attribute. It's mere existence indicates that the checkbox is checked, even with `checked="false"`. This code implies that the `includeServerHttpHeader` checkbox is not written correctly.

That being said, Selenium understands standard HTML checkboxes.
```suggestion
        boolean originalValue = checkbox.isSelected());
```

Reply:
> "checked" is a valueless attribute. It's mere existence indicates that the checkbox is checked, even with `checked="false"`. This code implies that the `includeServerHttpHeader` checkbox is not written correctly.
> 
> That being said, Selenium understands standard HTML checkboxes.

Thanks Trey. I also fixed up the other code that I patterned my change after.